README.md

```diff
@@ -14,14 +14,14 @@ very secret.
 
 | I want to... | Then you should... |
 | ------------ | ------------------ |
-| check out the site | `git clone https://github.com/department-of-veterans-affairs/vets-website.git` followed by `npm install`. Run `npm install` anytime `package.json` changes. | 
+| install the site | `git clone https://github.com/department-of-veterans-affairs/vets-website.git` followed by `cd vets-website` and `npm install`. Run `npm install` anytime `package.json` changes. | 
```

Comment:
Maybe lets say "clone the site". Saying install and then running `npm install` might be confusing.